cranelift/codegen/src/verifier/mod.rs

```diff
@@ -1036,12 +1042,9 @@ impl<'a> Verifier<'a> {
                         format!("{v} is defined by {block} which is not in the layout"),
                     ));
                 }
+                let user_block = self.func.layout.inst_block(loc_inst).unwrap();
```

Comment:
Let's use `expect()` here with a string indicating why this is infallible (we're traversing code that is in a block).